skt/runner.py

```diff
@@ -227,6 +227,46 @@ def getverboseresults(self, joblist):
 
         return result
 
+    def get_ltp_lite_logs(self, job_id):
+        """
```

Comment:
`pydocstyle` complains if there's no content in the first line of the comment, but we can fix that later if we ever enable it.